### DISCUSSION TITLE
Request clean build

### DISCUSSION THREAD

**Discussion by dogpoopy** [ROM-builders/temporary#23407] [General]
https://cirrus-ci.com/task/4640249758875648?logs=build#L3

Hi sir, may i request a clean build please. The compiler is targeting the wrong kernel path causing these bunch of "No such file or directory" error

And also there's seems to be an existing "Image" kernel  output in the build environment that cannot be overwritten or something (possible directory: /tmp/hi/out/target/product/vayu/obj/KERNEL_OBJ/arch/arm64/boot/Image base on our conversation here https://github.com/ROM-builders/temporary/discussions/22666) Please clear it if you may.

I think "make clobber" might be the fix if that's possible. Thank you!

**Comment by Apon77** [comment]
#22666 was done completely in clean directory. so if there is problem in #22666 then it is obviously from your side

**Reply by dogpoopy** [reply]
Would you please redo it again sir? I've track down the problem in #22666 and found out that the problem was caused by missing AndroidKernel.mk in my kernel source which i restored already. My last build included those fixes but i still have the same error and those "no such file or directory error" spam so i thought it would need a clean build.

**Reply by Apon77** [reply]
Clean done
